app/services/submissions_inviter.rb

```diff
@@ -5,7 +5,9 @@ def initialize(event_dates, event_venue)
   end
 
   def call(submissions)
-    if submissions.empty?
+    if !Setting.registration_period?
+      false
+    elsif submissions.empty?
```

Comment:
The idea was to to tell 'MailingsController' if the call was a success. if 'submissions' were empty, the result of 'submissions.each...' would be '[]' not 'false'  so I added another if clause.